src/Microsoft.Health.Fhir.Core/Models/FhirResourceFormat.cs

```diff
@@ -0,0 +1,14 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Health.Fhir.Core.Models
+{
+    public enum FhirResourceFormat
+    {
+        Xml = 1,
+        Json = 2,
+        Unknown = 3,
```

Comment:
Unknown should be the first item with 0

Reply:
Agree, these are the same as in Fhir.Stu3.ResourceFormat. Should we deviate?

Reply:
I think our code should follow our coding practices. I don't imagine us doing direct cast anyways if we need to convert so i don't think it would be a problem if the number is different?

Reply:
Sure